### PR TITLE
Update readme - V1 release

### PR DESCRIPTION
**Pull request by mindyyuan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,94 @@
...
-* TreeMap
+3. If you prefer, you can import all of Reactochart at once, though this may hinder some optimizations, such as webpack tree-shaking:
+  ```
+  import {XYPlot, XAxis, YAxis, LineChart} from 'reactochart';
+  ```
+or
+  ```
+  import * as Reactochart from 'reactochart';
+  ```
+4. See your first chart rendered!
```
Comment (krissalvador27) [inline README.md]: would you be able to include the code example with XYPlot wrapping LineChart?

**Comment by mindyyuan** [comment]
@krissalvador27 I added the LineChart example. Since you added links to the docs for each component, I'm wondering if we should still include the 'Live Examples' section? They link to the same pages, just one is local.

**Comment by krissalvador27** [comment]
I think it's fine the Live examples section is for those who want to run things locally.

Otherwise lgtm! Thanks! We will merge when I release v1

**Comment by krissalvador27** [comment]
realized we need to add instructions for importing the styles. will add that now and merge :)

**Pull request merged**